spec/hosts/nodes/tang01.dev.lsst.org_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'tang01.dev.lsst.org', :sitepp do
+  on_supported_os.each do |os, facts|
```

Comment:
It gets a bit confusing with this `facts` as local variable and the rspec specific `facts` in line 10. At Vox Pupuli we switched to:
```suggestion
  on_supported_os.each do |os, os_facts|
```

Reply:
Ya, that has been driving me crazy too...

Reply:
Uh, thanks...

```
 182 files changed, 520 insertions(+), 530 deletions(-)
```